src/fmu/dataio/_utils.py

```diff
@@ -350,14 +351,17 @@ def some_config_from_env(envvar="FMU_GLOBAL_CONFIG") -> dict:
     if envvar in os.environ:
         cfg_path = os.environ[envvar]
     else:
-        raise ValueError(
+        warnings.warn(
```

Comment:
Suggest to add to the warning a note that the ability to export without metadata may be deprecated in the future - it goes a bit against the whole purpose of fmu-dataio, does it not?